packages/react-sdk/src/pubsub/create.ts

```diff
@@ -10,6 +12,25 @@ export const createPubsub = <PublishMap>() => {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const emitter = createNanoEvents<Record<any, any>>();
 
+  type Task = () => void;
+  let rafHandler: ReturnType<typeof requestAnimationFrame>;
+  const tasks: Task[] = [];
+
+  const processArray = () => {
+    unstable_batchedUpdates(() => {
```

Comment:
why do you actually need this? why is just using raf no enough?

Reply:
discussed in dm, lets add a comment why we need to call batchedUpdates

Reply:
React 18 doesn't have unstable_batchedUpdates. All updates are batched.